spec/suspenders/generators/accessibility_generator_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+
+RSpec.describe Suspenders::AccessibilityGenerator, type: :generator do
+  describe "invoke" do
+    it "bundles the capybara_accessibility_audit gem" do
+      with_fake_app do
+        invoke! Suspenders::AccessibilityGenerator
+
+        expect("Gemfile").to have_bundled("install").matching(/capybara_accessibility_audit/)
+      end
+    end
+
+    it "bundles the citizensadvice/capybara_accessible_selectors GitHub repository gem" do
+      with_fake_app do
+        invoke! Suspenders::AccessibilityGenerator
+
+        expect("Gemfile").to have_bundled("install").matching(/capybara_accessible_selectors/)
+      end
+    end
```

Comment:
[Minor] How would you feel about [testing for syntax errors](https://github.com/thoughtbot/suspenders/blob/099493117c6eaf2f74bec958477c50cb63de96b1/spec/support/generator_matchers.rb#L27) in case there's a regression? We do that in several of our existing generator tests.

https://github.com/thoughtbot/suspenders/blob/099493117c6eaf2f74bec958477c50cb63de96b1/spec/suspenders/generators/inline_svg_generator_spec.rb#L8-L10